mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -93,11 +93,11 @@ def ocn_modelvsobs(config, field):
 
     elif field.lower() == 'sst':
 
-        ds = xr.open_mfdataset(infiles, preprocess=lambda x: preprocess_mpas(x, yearoffset=yr_offset,
-		                    timeSeriesStats=True,
-		                    timestr='time_avg_daysSinceStartOfSim',
-		                    onlyvars=['time_avg_activeTracers_temperature'],
-		                    selvals={'nVertLevels':1}))
+        ds = xr.open_mfdataset(infiles, preprocess=lambda x: 
+                               preprocess_mpas(x, yearoffset=yr_offset,
+                                               timestr=['xtime_start', 'xtime_end'],
+		                               onlyvars=['time_avg_activeTracers_temperature'],
+		                               selvals={'nVertLevels':1}))
         ds = remove_repeated_time_index(ds)
         ds.rename({'time_avg_activeTracers_temperature':'mpasData'}, inplace = True)
 
```

Comment:
@milenaveneziani, yes that's exactly right.  What I realized in the process of making this PR is that `timestr` will need to be a special case in the PR for addressing #20 because we want to support both the possibility of a single variable (e.g. `xtime` or `daysSinceStartOfSim`) and that of a pair of variable names like in this case.  That will be a bit trickier to implement, and may require tweaking the config reader to handle a list of lists (if it doesn't already) but it should be doable.  Again, this is a good case to run into now so we build in some more sophisticated functionality that will hopefully be useful for the future.